Xamarin.Forms.Core/DecorableTextElement.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+	[Flags]
+	[TypeConverter(typeof(TextDecorationConverter))]
+	public enum TextDecorations
+	{
+		None = 0,
+		Underline = 1 << 0,
+		Strikethrough = 1 << 1,
+	}
+	static class DecorableTextElement
+	{
+		public static readonly BindableProperty TextDecorationsProperty = BindableProperty.Create("TextDecorations", typeof(TextDecorations), typeof(IDecorableTextElement), TextDecorations.None);
```

Comment:
use `name(IDecorableTextElement.TextDecorations)`

Reply:
will change this, thanks